editor/src/components/canvas/remix/remix-utils.tsx

```diff
@@ -99,3 +128,405 @@ export function getRoutesFromRouteManifest(
     return []
   }
 }
+
+export function createAssetsManifest(routes: RouteManifest<EntryRoute>): AssetsManifest {
+  return {
+    entry: { imports: [], module: 'TODO' },
+    url: '/',
+    version: '1',
+    routes: routes,
+  }
+}
+
+export type RouteModuleWithFilePath = RouteModule & {
+  filePath: string
+}
+
+export interface RouteModulesWithFilePaths {
+  [routeId: string]: RouteModuleWithFilePath
+}
```

Comment:
Doesn't look like these two are used anymore?

Reply:
thanks! removed them in [192aafb](https://github.com/concrete-utopia/utopia/pull/4162/commits/192aafb3d2ab6c6136caf78643c15d6004f6ec02)